### PR TITLE
add mstest in cascade_clsaware

### PR DESCRIPTION
**Pull request by littletomatodonkey**
1. add mstest in cascadeclsaware, Based on cascade_rcnn_cls_aware_r101_vd_fpn model, coco mAP is improved from 43.9%(800x1333 test, standard nms ) 46.5%( ms test, the same method as `cascade_rcnn_r50_fpn_1x_ms_test.yml` )
2. remove unused code in cascade_rcnn.py
3. Solve the feature request in #293

### CODE REVIEW

File: ppdet/modeling/architectures/cascade_rcnn_cls_aware.py [hunk 1]
```diff
@@ -170,6 +170,98 @@ def build(self, feed_vars, mode='train'):
                 self.cascade_decoded_box, self.cascade_bbox_reg_weights)
             return pred
 
+    def build_multi_scale(self, feed_vars):
+        #         im = feed_vars['image']
```
Comment (jerrywgz) [inline ppdet/modeling/architectures/cascade_rcnn_cls_aware.py]: remove comment

**Comment by littletomatodonkey** [comment]
> 配置文件和MODEL ZOO这些一块更新下吧

测试时使用的是https://github.com/PaddlePaddle/PaddleDetection/blob/master/configs/cascade_rcnn_cls_aware_r101_vd_fpn_1x_softnms.yml

这个配置文件对应的模型，所以模型结构没有变，用户如果需要使用Cascade rcnn clsaware的mstest的话，按照mstest的方式修改yml文件即可~

File: ppdet/modeling/architectures/cascade_rcnn_cls_aware.py (hunk 1 above)
Comment (littletomatodonkey) [reply ppdet/modeling/architectures/cascade_rcnn_cls_aware.py]: done

**Comment by littletomatodonkey** [comment]
> config file and related doc are needed

fixed~

File: docs/MODEL_ZOO.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ The backbone models pretrained on ImageNet are available. All backbone models ar
 | SENet154-vd-FPN         | Faster         |     1     |  1.44x  |     3.408      |  42.9  |    -    | [model](https://paddlemodels.bj.bcebos.com/object_detection/faster_rcnn_se154_vd_fpn_s1x.tar) |
 | SENet154-vd-FPN         | Mask           |     1     |  1.44x  |     3.233      |  44.0  |  38.7   | [model](https://paddlemodels.bj.bcebos.com/object_detection/mask_rcnn_se154_vd_fpn_s1x.tar) |
 | ResNet101-vd-FPN            | CascadeClsAware Faster   |     2     |   1x    |     -     |  44.7(softnms)  |    -    | [model](https://paddlemodels.bj.bcebos.com/object_detection/cascade_rcnn_cls_aware_r101_vd_fpn_1x_softnms.tar) |
+| ResNet101-vd-FPN            | CascadeClsAware Faster   |     2     |   1x    |     -     |  46.5(mstest)  |    -    | [model](https://paddlemodels.bj.bcebos.com/object_detection/cascade_rcnn_cls_aware_r101_vd_fpn_1x_softnms.tar) |
```
Comment (jerrywgz) [inline docs/MODEL_ZOO.md]: 写成完整的multi-scale test吧，清晰些

Comment (littletomatodonkey) [reply docs/MODEL_ZOO.md]: 已修改

**Pull request merged**